### ISSUE TITLE
@AndroidParcelable to check before adding blank constructor

### ISSUE THREAD

**Issue by tobykurien** [tobykurien/Xtendroid#97]
Currently it forcefully adds a blank constructor and doesn't allow you to add your own

**Comment by Buggaboo** [comment]
I tried fixing this today, but for some reason, the compiler does not acknowledge the empty ctors that were generated by the [AndroidParcelable active annotation](https://github.com/Buggaboo/Xtendroid/tree/fixing_issues_96_97_98).

You may wanna have a go at it.

**Comment by tobykurien** [comment]
Not sure if we're missing each other - what I meant by this issue is
that if I create a bean and add an empty constructor, then add the
```@AndroidParcelable``` annotation, I get a compile error because there
are now 2 empty constructors. If ```@AndroidParcelable``` first checked
that the empty c'tor exists, then it can bypass adding it's own, and
hence no compile error :-)

**Comment by Buggaboo** [comment]
Yeah, I found it. Also, I found other issues. It's fixed. Patch coming up soon.
